lerna.json

```diff
@@ -1,5 +1,5 @@
 {
   "npmClient": "npm",
   "useWorkspaces": true,
-  "version": "1.0.0"
+  "version": "2.0.0-alpha.0"
```

Comment:
What does changing the version number here do? I thought it was automatically incremented when we publish to NPM.

Reply:
You’re right that it’s automatically incremented using semver as part of Lerna’s publishing workflow, but that calls `npm version` (or something analogous) under the hood. That command both increments this value and creates a git tag at the new version number.